### PR TITLE
Build fix

### PR DESCRIPTION
**Pull request by eed3si9n**
Continuation of #1841

### CODE REVIEW

File: build.sbt [hunk 1]
```diff
@@ -460,7 +477,7 @@ def otherRootSettings = Seq(
   Scripted.scriptedUnpublished <<= scriptedUnpublishedTask,
   Scripted.scriptedSource <<= (sourceDirectory in sbtProj) / "sbt-test",
   publishAll := {
-    (publishLocal).all(ScopeFilter(inAnyProject)).value
+    val _ = (publishLocal).all(ScopeFilter(inAnyProject)).value
```
Comment (jsuereth) [inline build.sbt:511]: why this change?

**Comment by jsuereth** [comment]
I think this now LGTM.

File: build.sbt (hunk 1 above)
Comment (eed3si9n) [inline build.sbt:511]: There was a warning saying the expression's not doing anything or something to that effect.

**Comment by eed3si9n** [comment]
Added two more commits related to the build.

**Comment by jsuereth** [comment]
LGTM
